common/main/en_SG.xml

```diff
@@ -148,7 +148,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 	<numbers>
 		<currencies>
 			<currency type="SGD">
-				<symbol>$</symbol>
+				<symbol>"S$"</symbol>
```

Comment:
The symbol should not have quote marks around it. But a bigger question: We currently have four locales for SG: en_SG, ms_SG, ta_SG, and zh_Hans_SG. All of them currently use just "S" for SGD, and for USD they have (either in the locale or inherited from parent) "US$" or "USD". We need a broader discussion about changing the symbol for SGD in all of the _SG locales.